view_tables.py

```
@@ -0,0 +1,36 @@
+        print(f"\nTable: {table_name}")
+        print("-" * 50)
+        
+        # Get table schema
+        cursor.execute(f"PRAGMA table_info({table_name});")
+        columns = cursor.fetchall()
+        print("\nColumns:")
+        for col in columns:
+            print(f"- {col[1]} ({col[2]})")
+        
+        # Get table contents
+        cursor.execute(f"SELECT * FROM {table_name}")
```

Ruff (0.13.3)

Possible SQL injection vector through string-based query construction

(S608)